packages/core/src/components/forms/label.md

```diff
@@ -19,12 +19,13 @@ below, clicking a label focuses its `<input>`.
 This component supports the full range of HTML props.
 
 ```tsx
-<Label
-    helperText="Helper text with details..."
-    text="Label A"
->
-    <input className="@ns-input" id="text-input" placeholder="Placeholder text" />
+<Label>
+    Label A
+    <input className="bp3-input" id="text-input" placeholder="Placeholder text" />
 </Label>
+
+<Label htmlFor="text-input">Label A</Label>
+<input className="bp3-input" id="text-input" placeholder="Placeholder text" />
```

Comment:
`className={Classes.INPUT}` please in both uses.

Reply:
Makes sense...done!